experiments/standalone/neural_snow/data_tools.jl

```diff
@@ -0,0 +1,362 @@
+using DataFrames, CSV, HTTP, Dates, StatsBase
+
+"""
+    data_url(id, state, fields)
+
+Return the url to gather SNOTEL data from a particular station.
+
+# Arguments
+- `id::String`: the id code of the station to access.
+- `state::String`: the state abbreviation of the station to access.
+- `fields::Vector{String}`: a list of strings specifying the SNOTEL parameters to collect.
+"""
```

Comment:
Maybe note the keyword arguments and what the default setting does.

What is the metric?